### ISSUE TITLE
Hyperparameter tune the LSTM

### ISSUE THREAD

**Issue by kdmalc** [kdmalc/fl-gestures#28]
Learning rate, num layers, num hidden nodes, inits, etc...

**Comment by kdmalc** [comment]
Ran for 3 days and did not finish... also performance was hovering around chance no matter what basically...

**Comment by kdmalc** [comment]
Irrelevant to current project goals.
